generator/src/main/java/org/corfudb/generator/Correctness.java

```diff
@@ -0,0 +1,25 @@
+package org.corfudb.generator;
+
+import ch.qos.logback.classic.Logger;
+import org.corfudb.runtime.object.transactions.TransactionalContext;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Created by rmichoud on 10/9/17.
+ */
+public class Correctness {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)